lua/entities/gmod_wire_expression2/core/custom/prop.lua

```diff
@@ -355,6 +355,41 @@ e2function string entity:propPhysicalMaterial()
 	return ""
 end
 
+hook.Add( "CanDrive", "checkPropStaticE2", function( ply, ent ) return ent.propStaticE2 == nil end )
+e2function void entity:propStatic( number static )
+	if not PropCore.ValidAction( self, this, "static" ) then return end
+	if static ~= 0 and this.propStaticE2 == nil then
+		local phys = this:GetPhysicsObject()
+		this.propStaticE2 = phys:IsMotionEnabled()
```

Comment:
Instead of
```
hook.Add( "CanDrive", "checkPropStaticE2", function( ply, ent ) return ent.propStaticE2 == nil end )
```
Do
```
hook.Add( "CanDrive", "checkPropStaticE2", function( ply, ent ) if not ent:GetUnFreezable() then return false end end )
```